plots/laws/thermodynamics/equations_of_state/van_der_waals_state_equation.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import numpy as np
+from sympy import symbols, Eq, solve
```

Comment:
CI is failing.

Reply:
After a day of searching, I still don't understand why this exception occurs.

Reply:
Fixed